### ISSUE TITLE
Personnaliser le message de la bêta avant l'envoi, avec la liste des commits pré-remplie

### ISSUE THREAD

**Issue by Eskimon** [zestedesavoir/zds-site#2532]
Lorsque la beta est mise à jour un nouveau message est posté dans son sujet de suivi sur le forum. Il serait cool que ce message liste les commits (s'ils ont un message) afin de rendre ce message un peu plus utile qu'un simple "up".

**Comment by Situphen** [comment]
Je sais pas. Les utilisateurs n'ont pas accès à l'historique donc pour mettre le message des commits, j'aimerais bien que l'on demande l'autorisation à l'auteur !

**Comment by pierre-24** [comment]
Je vois ce à quoi Eskimon veut arriver, et l'idée est pas si mauvaise que ça. À fortiori, il y a plusieurs manière d'imaginer une chose pareille, une d'entre elle serait qu'au moment de mettre en bêta (ou de la mettre à jour), une fenêtre s'ouvre avec une zone de texte. Par défaut, la zone de texte s'appellerai "résumé de vos modifications", et toujours par défaut, elle pourrait effectivement contenir la liste des messages de commit. L'auteur en fait ensuite ce qu'il veut ;)

(un peu comme ce qu'on fait quand on fait une pull request, quoi)

**Comment by Situphen** [comment]
@pierre-24 : Tu penses à une zone de texte qui contiendrai ce genre de texte ?

```
Bonjour à tous !

La beta du tutoriel a été mise à jour.

-> [[ lien-vers-la-béta ]] <-

[[ liste-des-commits ]]

Merci pour vos relectures
```

Ou ce qui est entre crochet serait remplacé par le back. Si l'utilisateur enlève le `[[ liste-des-commits ]]`, il n'y aura pas de liste de _commits_.

**Comment by pierre-24** [comment]
Non, mieux que ça. À chaque MàJ, t'as une zone de texte qui s'ouvre, et la liste des commits apparait en clair, genre 

```
Bonjour à tous !

La beta du tutoriel a été mise à jour.

Voici les changements:

- Création du chapitre machin
- Édition de l'extrait bidule
- Corrections orthographiques


Merci pour vos relectures
```

À l'auteur de virer la partie "voici les changements" si ça lui plait pas. Autrement dit, offrir à l'auteur d'écrire lui-même son message de MàJ, tout en lui en proposant un (c'est pour rejoindre ton point "moi, j'ai pas envie qu'on vois mes commits", bah tu peux les dégager si ça te plait pas ;) )

**Comment by pierre-24** [comment]
Ce système permet par ailleurs de virer les commits bateaux ou d'éditer les messages de commit le cas échéant (les messages par défaut étant très peu explicites et très ... Bateaux, justement)

**Comment by Situphen** [comment]
+9999

**Comment by Eskimon** [comment]
J'aime ce que tu proposes Pierre :) Ca rajoute une étape au process mais elle peut-être vite "évitée/acompli" et permettera d'avoir des mises à jour de beta un peu plus parlante :)

**Comment by Deuchnord** [comment]
Hello,

Je serais également intéressé par une telle fonctionnalité :)
Par contre, comme je le disais dans mon issue #3138, il serait intéressant d'avoir également quelque chose pour ajouter un commentaire sur le message initial du sujet lors de l'ouverture de la bêta, par exemple si l'on a une remarque à ajouter sur une partie du contenu :)
